README.md

```diff
@@ -123,6 +124,12 @@ The MDS specification is versioned using Git tags and [semantic versioning](http
 * [MDS Releases](https://github.com/openmobilityfoundation/governance/wiki/Releases) - current/recommended versions, timeline
 * [Release Guidelines](https://github.com/openmobilityfoundation/governance/blob/main/technical/ReleaseGuidelines.md)
 
+## Data Validation
+
+To help with MDS data and feed validation, please see our OpenAPI schema description in the OMF [mds-openapi](https://github.com/openmobilityfoundation/mds-openapi) repository. Browsable interactive documentation is also linked to in that repository.
+
+Starting with MDS 2.0, [schema](/schema) validation of endpoints is done using OpenAPI, instead of the former JSON Schema.
```

Comment:
```suggestion
Starting with MDS 2.0, OpenAPI documents will describe MDS endpoints and allow for [schema](/schema) validation, expanding on the JSON Schemas formerly housed in this repository.
```

Reply:
Just clarifying - all the JSON Schema stuff is still here, it's now just wrapped in OpenAPI documents.